antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/SqlBase.g4

```diff
@@ -695,7 +695,7 @@ dateExpression
     ;
 
 encoding
-    : PLAIN | PLAIN_DICTIONARY | RLE | DIFF | TS_2DIFF | GORILLA | REGULAR
+    : PLAIN | PLAIN_DICTIONARY | RLE | DIFF | TS_2DIFF | GORILLA | REGULAR | DICTIONARY
```

Comment:
I suggest renaming the original `PLAIN_DICTIONARY` as `DICTIONARY`, and modify the relevant documents, including `Format.md` and so on...